test/e2e-common.sh

```diff
@@ -315,12 +315,40 @@ function install_consolidated_channel_crds {
   wait_until_pods_running "${SYSTEM_NAMESPACE}"
 }
 
+function install_released_consolidated_source {
```

Comment:
I would remove `consolidated`. Just  `install_released_source`. 

I suppose the upgrade tests are black-box tests, in which case they can also run against the multi-tenant source variant.

Reply:
The function name contained `consolidated` word before, that's why I didn't remove it.

That's right those tests should be a black box, but I believe future conditional logic should be in golang in installation package rather than bash scripts.

Reply:
My two cents: After reading `install_released_consolidated_source` my first question is - what "source" ? It could be "source code". Maybe we could say `install_released_event_source` instead.